templates/zerver/help/configure-notification-bot.md

```diff
@@ -1,21 +1,83 @@
-# Configure notification bot
+# Notification bot
+
+The Zulip notification bot automatically generates messages for
+various organization level events, including:
+
+* Stream settings changes (such as [name](/help/rename-a-stream),
+  [description](/help/change-the-stream-description),
+  [permission](/help/stream-permissions) and
+  [policy](/help/stream-sending-policy) updates)
+* A topic being [resolved/unresolved](/help/resolve-a-topic)
+* New stream and new user announcements
+
+Organization administrators can configure where (and whether)
+[new stream](#new-stream-notifications) and
+[new user](#new-user-notifications) announcement messages are sent,
+as well as the [language](#notifications-language) that all general
+notification bot messages are sent in.
+
+## Configure notification bot
 
 {!admin-only.md!}
 
-The Zulip notification bot sends a message whenever:
+### New stream notifications
+
+When a new public stream is [announced][new-stream-options], you can
+configure where the Zulip notification bot sends a message, or disable
+the new stream notification messages entirely.
+
```

Comment:
Since you don't *configure* this when the stream is announced, let's restructure the sentence to something like:

> You can configure where the Zulip notification bot [announces][new-stream-options] new public streams, or disable the new stream notification messages entirely.

And similarly for new user notifications below.